apps/fabric-website/src/pages/Styles/LayoutPage/LayoutPage.tsx

```diff
@@ -32,7 +32,7 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
           content: (
             <>
               <p>
-                Fabric comes with a mobile-first, 12-column, responsive grid that you can use to create flexible layouts for a variety of
+                Fluent UI comes with a mobile-first, 12-column, responsive grid that you can use to create flexible layouts for a variety of
```

Comment:
This page needs to continue to use Fabric, since it's only relevant to Fabric Core.

Reply:
Should probably also update it to specifically say "Fabric Core comes with..."

Reply:
```suggestion
                Fabric Core comes with a mobile-first, 12-column, responsive grid that you can use to create flexible layouts for a variety of
```